pkg/operator/staticpod/controllers.go

```diff
@@ -63,7 +73,7 @@ func NewControllers(targetNamespaceName, staticPodName, podResourcePrefix string
 		staticPodOperatorClient,
 		kubeClient,
 		eventRecorder,
-	)
+	).WithInstallerPodMutationFn(installerPodMutationFn)
```

Comment:
alternatively turn this into another `WithInstallerPodMutationFn` func against `staticPodOperatorControllers`. We are clearly getting too many parameters. `With*` funcs will help to keep the usual case shorter.

Reply:
> alternatively turn this into another `WithInstallerPodMutationFn` func against `staticPodOperatorControllers`. We are clearly getting too many parameters. `With*` funcs will help to keep the usual case shorter.

hrm, no struct.  Can I get an IOU?